R/leaflet.R

```diff
@@ -70,3 +102,71 @@ mapOptions <- function(map, zoomToLimits = c("always", "first", "never")) {
 
   map
 }
+
+# CRS classes supported
+crsClasses <- list( 'L.CRS.EPSG3857', 'L.CRS.EPSG4326', 'L.CRS.EPSG3395',
+                    'L.CRS.Simple', 'L.Proj.CRS', 'L.Proj.CRS.TMS' )
```

Comment:
Yes, but on a side note this is not an exported list, it is used internally for checking the crsClass argument to mapOptions. So technically it can change w/o breaking API.
But just to be sure, I checked 1.0 code and these projs. are indeed supported there.